Pages/WishlistManagement/wishlist.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DiagonAlley-Wishlist</title>
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="/Stylesheets/PageStyles/wishlist.css">
```

Comment:
Move the title below the stylesheets.

Reply:
Hey I have kept the titles in all pages at the top, if that format is not good I will change them.